gnome-initial-setup/gis-factory-dialog.c

```diff
@@ -82,17 +84,11 @@ static void
 show_error (GisFactoryDialog *self,
             const GError     *error)
 {
-  GtkWidget *dialog;
-
-  dialog = gtk_message_dialog_new (GTK_WINDOW (self),
-                                   GTK_DIALOG_MODAL | GTK_DIALOG_DESTROY_WITH_PARENT,
-                                   GTK_MESSAGE_ERROR,
-                                   GTK_BUTTONS_CLOSE,
-                                   "%s", error->message);
-  g_signal_connect (dialog, "response",
-                    G_CALLBACK (gtk_window_destroy),
-                    NULL);
-  gtk_window_present (GTK_WINDOW (dialog));
+  g_autoptr(GtkAlertDialog) dialog = NULL;
+
+  dialog = gtk_alert_dialog_new (_("Failed to Enter Test Mode"));
```

Comment:
I deeply dislike this capitalisation but that's what the GNOME HIG mandates.

https://developer.gnome.org/hig/guidelines/writing-style.html#capitalization